packages/console/src/components/HoverableTextWrapper/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { conditional } from '@silverhand/essentials';
+import classNames from 'classnames';
+import { useRef } from 'react';
+
+import { Tooltip } from '@/ds-components/Tip';
+import useTextOverflow from '@/hooks/use-text-overflow';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  children: string;
+};
+
+function HoverableTextWrapper({ children }: Props) {
```

Comment:
should we just use:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title